_data/simple-icons.json

```diff
@@ -5525,6 +5525,11 @@
             "hex": "F7B500",
             "source": "https://www.sketch.com/press/"
         },
+        {
+            "title": "SketchUp",
+            "hex": "E9292F",
```

Comment:
I'm not sure about this color?
Where did you get it from, exactly?

The color of the Wikimedia icon is different, and so is the icon on the website... (`#F62737`) 🤔